README.md

```diff
@@ -1,25 +1,65 @@
 # presentation-microservices-kotlin
-Presentation code and material for micro-services with kotlin
+
+Build microservices wth Kotlin + Spring Boot
+Presented at GDG Kotlin/Everywhere Hong Kong : A Day of Kotlin, 31st Aug 2019
+Slides can be found here
```

Comment:
missing link to `here`, consider convert it to `slide` file, put on https://github.com/oursky/slides and link here?

Reply:
can do, let's have a chat over this on Monday?